src/components/CalendarGrid.vue

```diff
@@ -220,6 +220,19 @@ export default {
 			}
 		}, 1000)
 
+		// Trigger the select event programmatically on initial page load to show the new event
+		// in the grid.
+		this.$nextTick(() => {
+			if (['NewPopoverView', 'NewSidebarView'].includes(this.$route.name)) {
+				const calendarApi = this.$refs.fullCalendar.getApi()
+				calendarApi.select({
+					start: new Date(parseInt(this.$route.params.dtstart) * 1000),
```

Comment:
No handling for when an invalid date is created?

Reply:
FC has an internal sanity check and does not propagate the select event to our handler if the dates are invalid. I just tested this.

Reply:
PS: I still committed the check because why not :D